app/build.gradle

```diff
@@ -5,12 +5,11 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.2"
+    compileSdkVersion 28
     defaultConfig {
         applicationId "com.example.reflector_android"
-        minSdkVersion 26
-        targetSdkVersion 29
```

Comment:
Something to note but you also went down in this too. No need to do anything here. Just bringing it to your attention.